### PR TITLE
Desktop- version

### PR DESCRIPTION
**Pull request by MILINDI7**
## What?
After creating and styling the mobile version, in this new branch, I updated the portfolio to be supported with different device screens due to media queries. 

## Why?
Though It is relevant that the web should be viewed well onto different device's screen, It is also in the project requirements.

## How?
By the use of media queries, contents are adjusted how to be viewed on different view ports.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -158,19 +166,34 @@ <h5>Skills</h5>
...
+          <label>
+            <input id="texts" type="text" name="names" placeholder="Full name" maxlength="30" required>
+          </label>
+          <label>
+            <input id="mails" type="email" name="email" placeholder="Email address" required>
+          </label>
+          <label>
+            <textarea name="message" maxlength="500" required>Enter text here</textarea>
+          </label>
+          <button class="form-submit" type="submit">Get in touch</button>
```
Comment (emmiiorji) [inline index.html:181]: Well done for a good design. Kindly include the button interactions as specified on the figma design.

Given interactions:
image

File: style.css [hunk 2]
```diff
@@ -785,3 +785,227 @@ textarea {
...
+  #hh1 {
+    display: none;
+  }
+
+  .works-heading img {
+    display: none;
+  }
+
+  .cards {
+    width: 1171px;
```
Comment (emmiiorji) [inline style.css:880]: Kindly ensure that the website is responsive across screen sizes and not for a width. You have in many cases assigned fixed widths to elements and this way will most times cause the website to only work for a specific screen size. Right now, you can see that there's an overflow on the website.

For a better user experience, kindly avoid a horizontal overflow.

image

File: style.css [hunk 3]
```diff
@@ -785,3 +785,227 @@ textarea {
...
+@media only screen and (min-width: 768px) {
+  body {
+    width: 1340px;
+  }
+
+  main {
+    width: 100%;
+    margin: 0;
+  }
+
```
Comment (emmiiorji) [inline style.css:812]: Kindly ensure that the mobile version (375px or wider) is responsive until the desktop screen size (768px or wider).

Your design at 598px:
image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @MILINDI7, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

A little bit more

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Proper use of GitHub flow


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (MILINDI7) [reply index.html:181]: Please check that this was supposed to be tackled after this project is approved, It is not required in the current one.

Comment (emmiiorji) [reply index.html:181]: Hi @MILINDI7,

Thanks for your response. It's a design requirement which is expected to be followed closely.

Comment (Nyame-Wolf) [inline index.html:181]: I suggest that just as earlier advised, kindly Kindly include the button interactions as specified on the figma design. Its part of the project requirements, from the mobile design.

File: style.css [hunk 4]
```diff
@@ -785,3 +785,227 @@ textarea {
...
+  #last {
+    display: none;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  body {
+    width: 1300px;
```
Comment (Nyame-Wolf) [inline style.css:805]: As earlier indicated, I suggest you fix the margins and widths of your application. 
Your app seems to be leaning towards the left, its more to the left and the width is inconsistent to the figma template.
This is your design:
Screenshot 2023-01-20 at 10 21 12
Screenshot 2023-01-20 at 10 22 15
This is the figma tempate:
Screenshot 2023-01-20 at 10 21 06
Screenshot 2023-01-20 at 10 22 29

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi @MILINDI7  :wave: 

Your project page is looking fantastic so far :star_struck: ! 

However, there is still some room for improvement, that can help you have better code and make your project professional :sunglasses: 



## Highlights :sparkles:

- Great work using the right GitHub Flow :100: 
- Great work using the  Figma design ☑️ 
- Linters are passing
- Project Documentation (PR, README) is professional :necktie: 

Congrats on your work so far! :clap:  

## Required Changes ♻️

There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**